src/main/java/jdbchomework/console/AbstractMenu.java

```diff
@@ -0,0 +1,56 @@
+package jdbchomework.console;
+
+import jdbchomework.MainWithJdbc;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.List;
+
+public abstract class AbstractMenu {
+
+    public static final String CHOOSE_ACTION = "Choose action: ";
+    public static final String ERROR_INCORRECT_MENU_ITEM_SELECTED = "ERROR: Incorrect menu item selected \n";
+    public static final String SPLIT_LINE = "**************************************************************";
+    protected MenuContents menuContents;
+    private org.slf4j.Logger log = LoggerFactory.getLogger(this.getClass());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the "log" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1450)